### ISSUE TITLE
Change contact?

### ISSUE THREAD

**Issue by daviddelmonte** [yudiz-solutions/KPContactPicker#1]
Hello, using your control, is there a way to change contact information? Thanks

**Comment by ravi-yudiz** [comment]
No, there is no way to change contact information. Feel free to contribute if you want. thanks.
